webapp/public/pdf-lib/pdf-lib.min.js

```diff
@@ -0,0 +1,15 @@
+!function(t,e){"object"==typeof exports&&"undefined"!=typeof module?e(exports):"function"==typeof define&&define.amd?define(["exports"],e):e((t=t||self).PDFLib={})}(this,(function(t){"use strict";
+/*! *****************************************************************************
+    Copyright (c) Microsoft Corporation. All rights reserved.
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use
```

Comment:
Just double checking: Apache License 2.0 is okay? Weirdly the github for pdf-lib says MIT license but this indicates a different one. A quick google makes it seem like they're pretty comparable, but thought it might be worth calling out, just in case.

Also, do we need a LICENSE file? I see that in some of the other dependencies we've pulled into this location, but not sure if it's actually required.

Reply:
Hm I didn't copy it over as I didn't see it in zip-js, but I'll throw one in if we want?

Apache's fine, the reason this is different than pdf-lib is because this is the declaration for the dependency of zip-js (notice it's for a msft dependency in the first place, where pdf-lib is from a github employee https://github.com/Hopding/pdf-lib ~)

Reply:
Eh, no worries if we don't need the LICENSE file. Sounds like a non-issue. Thanks for the info!